kotlinx.fuzz.engine/src/main/kotlin/kotlinx/fuzz/log/StdoutLogger.kt

```diff
@@ -0,0 +1,120 @@
+
+    override fun isTraceEnabled(marker: Marker?): Boolean = isLevelEnabled(Level.TRACE)
+    override fun trace(marker: Marker?, msg: String?) = trace(msg)
+    override fun trace(marker: Marker?, format: String?, arg: Any?) = trace(format, arg)
+    override fun trace(
+        marker: Marker?,
+        format: String?,
+        arg1: Any?,
+        arg2: Any?,
+    ) = trace(format, arg1, arg2)
+    override fun trace(marker: Marker?, format: String?, vararg arguments: Any?) = trace(format, *arguments)
+    override fun trace(marker: Marker?, msg: String?, t: Throwable?) = trace(msg, t)
+
+    override fun isDebugEnabled(): Boolean = isLevelEnabled(Level.DEBUG)
+    override fun debug(msg: String?) = log(Level.DEBUG, msg)
+    override fun debug(format: String?, arg: Any?) = log(Level.DEBUG, format?.format(arg))
+    override fun debug(format: String?, arg1: Any?, arg2: Any?) = log(Level.DEBUG, format?.format(arg1, arg2))
+    override fun debug(format: String?, vararg arguments: Any?) = log(Level.DEBUG, format?.format(*arguments))
+    override fun debug(msg: String?, t: Throwable?) = log(Level.DEBUG, msg, t)
+
+    override fun isDebugEnabled(marker: Marker?): Boolean = isLevelEnabled(Level.DEBUG)
+    override fun debug(marker: Marker?, msg: String?) = debug(msg)
+    override fun debug(marker: Marker?, format: String?, arg: Any?) = debug(format, arg)
+    override fun debug(
+        marker: Marker?,
+        format: String?,
+        arg1: Any?,
+        arg2: Any?,
+    ) = debug(format, arg1, arg2)
+    override fun debug(marker: Marker?, format: String?, vararg arguments: Any?) = debug(format, *arguments)
+    override fun debug(marker: Marker?, msg: String?, t: Throwable?) = debug(msg, t)
+
+    override fun isInfoEnabled(): Boolean = isLevelEnabled(Level.INFO)
+    override fun info(msg: String?) = log(Level.INFO, msg)
+    override fun info(format: String?, arg: Any?) = log(Level.INFO, format?.format(arg))
+    override fun info(format: String?, arg1: Any?, arg2: Any?) = log(Level.INFO, format?.format(arg1, arg2))
+    override fun info(format: String?, vararg arguments: Any?) = log(Level.INFO, format?.format(*arguments))
+    override fun info(msg: String?, t: Throwable?) = log(Level.INFO, msg, t)
+
+    override fun isInfoEnabled(marker: Marker?): Boolean = isLevelEnabled(Level.INFO)
+    override fun info(marker: Marker?, msg: String?) = info(msg)
+    override fun info(marker: Marker?, format: String?, arg: Any?) = info(format, arg)
+    override fun info(
+        marker: Marker?,
+        format: String?,
+        arg1: Any?,
+        arg2: Any?,
+    ) = info(format, arg1, arg2)
+    override fun info(marker: Marker?, format: String?, vararg arguments: Any?) = info(format, *arguments)
+    override fun info(marker: Marker?, msg: String?, t: Throwable?) = info(msg, t)
+
+    override fun isWarnEnabled(): Boolean = LoggerFacade.LOG_LEVEL.toInt() <= Level.WARN.toInt()
```

Comment:
In `LoggerWrapper` you have `isLevelEnabled` function. Seem a good place to use it